drivers/hmis/app/models/hmis/form/assessment_processor.rb

```diff
@@ -66,28 +67,30 @@ def enrollment_coc_factory(create: true)
       )
   end
 
+  def common_attributes(create: true, id_field_name: nil)
+    attributes = {}
+    attributes[:data_collection_stage] = assessment_detail.data_collection_stage
+    attributes[:personal_id] = enrollment_factory.client.personal_id
+    attributes[:information_date] = assessment_detail.assessment.assessment_date
+    attributes[:user_id] = assessment_detail.assessment.user_id
+    attributes[:date_updated] = DateTime.current
+    attributes[:date_created] = DateTime.current if create
+    attributes[id_field_name] = Hmis::Hud::Base.generate_uuid if create && id_field_name.present?
```

Comment:
@awisema we need to set all these fields for most of the records - does this refactor look OK to you? 
Should probably update so that we call `DateTime.current` once and use the same value for all records.